sandbox/extension/Humboldt_2023-09-01.xml

```diff
@@ -0,0 +1,87 @@
+
+    <!-- Material Collected -->
+    <property group='Material Collected' name='hasVouchers' namespace='http://rs.tdwg.org/eco/terms/' qualName='http://rs.tdwg.org/eco/terms/hasVouchers' dc:relation='https://tdwg.github.io/eco/terms/#eco:hasVouchers' dc:description='Specimen vouchers were collected during the dwc:Event.' comments='' examples='`true`; `false`' required='false'/>
+    <property group='Material Collected' name='voucherInstitutions' namespace='http://rs.tdwg.org/eco/terms/' qualName='http://rs.tdwg.org/eco/terms/voucherInstitutions' dc:relation='https://tdwg.github.io/eco/terms/#eco:voucherInstitutions' dc:description='A list (concatenated and separated) of the names or acronyms of the institutions where vouchers collected during the dwc:Event were deposited.' comments='Recommended best practice is to separate multiple values in a list with space vertical bar space ( | ).' examples='`FMNH`; `AMNH | MVZ`; `Nairobi National Museum`' required='false'/>
+    <property group='Material Collected' name='hasMaterialSamples' namespace='http://rs.tdwg.org/eco/terms/' qualName='http://rs.tdwg.org/eco/terms/hasMaterialSamples' dc:relation='https://tdwg.github.io/eco/terms/#eco:hasMaterialSamples' dc:description='Material samples were collected during the dwc:Event.' comments='' examples='`true`; `false`' required='false'/>
+    <property group='Material Collected' name='materialSampleTypes' namespace='http://rs.tdwg.org/eco/terms/' qualName='http://rs.tdwg.org/eco/terms/materialSampleTypes' dc:relation='https://tdwg.github.io/eco/terms/#eco:materialSampleTypes' dc:description='A list (concatenated and separated) of material sample types collected during the dwc:Event.' comments='Recommended best practice is to use a controlled vocabulary and separate multiple values in a list with space vertical bar space ( | ).' examples='`whole organism`; `skeleton`; `tissue | blood | fecal | stomach content`' required='false'/>
+
+    <!-- Sampling Effort -->
+    <property group='Sampling Effort' name='samplingPerformedBy' namespace='http://rs.tdwg.org/eco/terms/' qualName='http://rs.tdwg.org/eco/terms/samplingPerformedBy' dc:relation='https://tdwg.github.io/eco/terms/#eco:samplingPerformedBy' dc:description='A person, group, or organization responsible for recording the dwc:Event.' comments='The sampling dwc:Event could be at any level of hierarchy. In the case of a higher level (parent) dwc:Event, include all the organizations or people involved in the child dwc:Events that contributed to the parent dwc:Event. Recommended best practice is to separate multiple values in a list with space vertical bar space ( | ).' examples='`North American Butterlfy Association`; `KK Wall`; `JJ Green`; `LL Pink and FF Grey | Aspen Center for Environmental Studies`' required='false'/>
```

Comment:
`samplingPerformedByID` was expected to accompany this to be consistent with recordedBy and identifiedBy and in keeping with the global movement towards person identifiers.

Should I open an issue in the public commentary?